src/openrct2/entity/Staff.cpp

```diff
@@ -388,6 +401,52 @@ bool Staff::HasPatrolArea() const
     return std::any_of(std::begin(PatrolInfo->Data), std::end(PatrolInfo->Data), hasData);
 }
 
+std::vector<TileCoordsXY> Staff::GetPatrolArea() const
+{
+    std::vector<TileCoordsXY> area;
+    if (PatrolInfo != nullptr)
```

Comment:
I know its just preference but I think that early exits makes code slightly easier to read, each scope adds a lot of empty space and generally speaking if every condition to bail early is tested right at the beginning it also makes the logic easier to grasp. Just a suggestion